templates/templates/orchestrator.template

```diff
@@ -0,0 +1,22 @@
+{%- if manifest.dbs %}
```

Comment:
ditto on moving to a subdir for preview templates

Reply:
changed